email/app.py

```diff
@@ -25,22 +42,13 @@ def send_email():
     # pobierac recipients i variables z post body bo to array emaili
     recipients = ['test@gmail.com']
     templateName = 'test'
-    variables = [{exampleGithubLogin:'Tester mikroserwisu'}]
+    variables = [{'exampleGithubLogin':'Tester mikroserwisu'}]
 
-    service = MJMLHandler(get_ip_for_service('mjml')['ip'])
+    # service = MJMLHandler_converter(get_ip_for_service('mjml')['ip'])
     # trzeba sprawdzić jak działa
-
-    _sender = EmailSender(STMPConfig)
-    _sender.connect()
-
-    for idx, recipient in enumerate(recipients, 1):
-        converted_mail = service.convert(templateName, variables)
-
-        res_email = _sender.send_mail(recipients=recipient, html=data['templateHtml'])
-        print('sent', idx,  '/', len(recipients))            
     
-    _sender.disconnect()
-
+    result = send_emails.delay(recipients=recipients)
+    result.wait()
```

Comment:
chyba wywoływanie rzeczy z kolejki powinno być niezależnie postawione a nie w endpoincie
i powinno działać na zasadzie, jeśli cokolowiek się pojawi -> wykonaj

Reply:
to nie jest kolejka, to jest rozdysponowanie zadań do asynchronicznych workerów, które wykonują te zadania, kolejka jest tu raczej nie potrzebna patrząc na to, że wysłanie maila trwa jakieś 5s, chyba, że źle się zrozumieliśmy

Reply:
tzn ja widzę to tak, aby była sobie kolejka z taskami, a akcja "wysyłka maila xyz..." była jednym z tasków, /send-mail wrzuca task do kolejki, która ma wbudowane zdejmowanie automatycznie wszystkich zakolejkowanych zadań